### ISSUE TITLE
v5.7.0 release doesn't contain authenticators in assets

### ISSUE THREAD

**Issue by fm100** [kubernetes-sigs/aws-iam-authenticator#449]
All the other releases contain `aws-iam-authenticator-{VERSION}-{PLATFORM}-amd64` files but `v5.7.0` does not. Is it intended change for the release?

**Comment by jshields** [comment]
FYI, this is breaking https://circleci.com/developer/orbs/orb/circleci/aws-eks

**Comment by j-carvalho** [comment]
> FYI, this is breaking https://circleci.com/developer/orbs/orb/circleci/aws-eks Seems the build assets are missing for release v5.7.0, which results in a 404 from our build pipeline.

as a fallback, it seems that the orb has this property authenticator-release-tag, that you can use to set the previous release version of this lib

**Comment by milosivanovic** [comment]
This has happened before: https://github.com/kubernetes-sigs/aws-iam-authenticator/issues/344

**Comment by milosivanovic** [comment]
@j-carvalho I tried this, but it doesn't appear to make any difference and still fails with a 404. Were you able to get it working? Here's what I added: https://github.com/CircleCI-Public/aws-eks-orb/issues/62#issuecomment-1093381864

**Comment by felipewatanabe-openco** [comment]
@j-carvalho, it worked for me, thanks. @milosivanovic here is part of my config.yaml, maybe could help.

```
  deploy-app:
    description: |
      Installs a helm chart into the EKS cluster.
      Requirements: helm should be installed on the cluster.
    executor: aws-eks/python3
    parameters:
      authenticator-release-tag:
        default: v0.5.6
        description: |
          Specifies which release-tag version of the authenticator to install.
        type: string
```

**Comment by milosivanovic** [comment]
I just noticed that the v is hardcoded in the orb: `v${VERSION}` source: https://github.com/CircleCI-Public/aws-eks-orb/blob/6ca33e02f984ea02d6b37c974a9edc502e0a7c8a/src/scripts/install-aws-iam-authenticator.sh#L21

Let me try without the v.

**Comment by paulosman** [comment]
@milosivanovic We were able to get the release-tag working as well. We're using aws-eks-orb@1.1.0 and it works for us. Relevant bits from our circle config:

```
    steps:
      - kubernetes/install-kubectl: *kubectl-install-params
      - aws-eks/update-kubeconfig-with-authenticator:
          authenticator-release-tag: v0.5.6
          aws-region: $AWS_REGION
          cluster-name: << parameters.cluster-name >>
          install-kubectl: false
```

Docs on the circle site: https://circleci.com/developer/orbs/orb/circleci/aws-eks#commands-update-kubeconfig-with-authenticator

**Comment by fm100** [comment]
It worked for me too.

`authenticator-release-tag: "v0.5.6"`

**Comment by jacobleesinger** [comment]
did not work for me, I am also getting `/bin/bash: line 13: v0.5.6: invalid variable name`

**Comment by wasims1** [comment]
can someone share the full config of circleci/aws-eks including the version? i'm using circleci/aws-eks@2.1.1 and still getting `invlaid variable name` like many others....

**Comment by russellraed** [comment]
Adding release tag v0.5.6 did not work for us as well. However, so we had to manually install aws-iam-authenticator:
```- run:
          command: |
            curl -o aws-iam-authenticator https://s3.us-west-2.amazonaws.com/amazon-eks/1.21.2/2021-07-05/bin/linux/amd64/aws-iam-authenticator
            chmod +x ./aws-iam-authenticator
            mkdir -p $HOME/bin && cp ./aws-iam-authenticator $HOME/bin/aws-iam-authenticator && export PATH=$PATH:$HOME/bin
            echo 'export PATH=$PATH:$HOME/bin' >> ~/.bashrc
            aws-iam-authenticator help
            curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
            sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
            kubectl version --client
```
Add this before aws-eks/update-kubeconfig-with-authenticator step in circleci

**Comment by wasims1** [comment]
to anyone still having this issue, the aws-eks orb itself issued an emergency update just change the version to :2.1.2 and it should work for now...

**Comment by johan-lejdung** [comment]
@wasims1 that fixed it for me. Although I had to update some syntax since I were still on a beta version of that orb 😅

**Comment by sartor1** [comment]
@[wasims1](https://github.com/wasims1) you've probably already fixed the issue, but try `authenticator-release-tag: "0.5.6"`, it should work. 'Cause "v" letter is hard-coded in the job with download-url: `DOWNLOAD_URL="https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v${VERSION}/${FILENAME}_${VERSION}_${PLATFORM}_amd64"
`

**Comment by nckturner** [comment]
I'm going to assume this is fixed because I see assets on 0.5.7.
